### PR TITLE
Added Contact support api

### PR DESCRIPTION
**Pull request by Shyam-Khokhariya**

### CODE REVIEW

File: src/utils/HttpRequests.js [hunk 1]
```diff
@@ -104,3 +104,13 @@ export const getPendingConversionCount = async (address) => {
     throw error;
   }
 };
+
+export const submitFeedback = async (params) => {
+  try {
```
Comment (raamb) [inline src/utils/HttpRequests.js:109]: Pull URL from env, should not be hardcoded

File: src/pages/Contact/hooks/contactSupportHook.js [hunk 2]
```diff
@@ -45,28 +47,51 @@ const useContactSupportHook = () => {
...
+      email,
+      phone_no: '',
+      message_type: type,
+      subject: '',
+      message,
+      attachment_details: {}
+    };
+    try {
+      const data = await submitFeedback(params);
+      if (data.status === 'success') {
```
Comment (rajeevbbqq) [inline src/pages/Contact/hooks/contactSupportHook.js:74]: do we need this `if` check, it won't make any state update if the API is throwing errors

File: src/pages/Contact/hooks/contactSupportHook.js [hunk 3]
```diff
@@ -45,28 +47,51 @@ const useContactSupportHook = () => {
...
-        setShowSuccessMessage(false);
-      }, 10000);
+    setIsLoading(true);
+    const params = {
+      source: 'BRIDGE',
+      name,
+      address: '',
+      email,
+      phone_no: '',
+      message_type: type,
```
Comment (rajeevbbqq) [inline src/pages/Contact/hooks/contactSupportHook.js:67]: where is the definition for variable `type` ?

File: src/utils/HttpRequests.js [hunk 4]
```diff
@@ -104,3 +104,13 @@ export const getPendingConversionCount = async (address) => {
...
+    const data = await axios.post('https://huvfmeboq0.execute-api.us-east-1.amazonaws.com/rt-v2/user/message', params);
```
Comment (rajeevbbqq) [inline src/utils/HttpRequests.js:110]: can you move this API endpoint to .env ? For mainnet the endpoint will be different

File: src/pages/Contact/hooks/contactSupportHook.js (hunk 3 above)
Comment (Shyam-Khokhariya) [reply src/pages/Contact/hooks/contactSupportHook.js:67]: const [type, setType] = useState('Bug'); line 11

**Review by rajeevbbqq — APPROVED** [review]

**Pull request merged**